docs/background_info/extended_models_optimization.md

```diff
@@ -0,0 +1,209 @@
+If a model or topic contains extended classes, the inclusive base classes are implemented in the physical database. The users only want to see, what is relevant for them and mostly work on the most extended instance of the topics/classes. Model Baker detects the ***irrelevant*** tables and offers optimization strategies to the users on ***smart1intehritance*** implementations.
+
+In the [workflow wizard](../../user_guide/import_workflow/#optimize-qgis-project-if-extended) you can choose the optimization [strategy]([corresponding chapter](../../background_info/extended_models_optimization/#strategies) and receive a nicely prepared layertree and forms.
+
+But what is done in the backend?
+
+## Backend Sollution
+
+### Assumptions
+
+Since it's impossible to care for all the cases,  we need to make some assumptions what mostly would be the use case.
+
+- When you extend a base class with the ***same name***, you intend to "replace" it, otherwise you would rename it.
+- When you extend a base class ***multiple times*** (what you do with different names) then you intend to "replace" it.
+- Exception for the two cases above: When you extended the class it in the ***same model*** but another topic (because if you intent to "replace" it, you would have made it `ABSTRACT`).
```

Comment:
```suggestion
- Exception for the two cases above: When you extended the class in the ***same model*** but another topic (because if you intent to "replace" it, you would have made it `ABSTRACT`).
```